Wrong error about @GetterName?

I'm seeing this trying to build my project:
```
MembershipData.java:81: error: Kotshi: Could not find a getter named getIs_premium, annotate the parameter with @GetterName if you use @JvmName
    com.chess.net.model.ExpirationInfo date, int is_premium, int level, @org.jetbrains.annotations.NotNull()
                                                 ^
```
Here is my code:
```
@JsonSerializable
class MembershipItem(
    override val data: MembershipData
) : BaseResponseItem<MembershipData>(data = data)

/* SAMPLE JSON
  "date": {
        "expires": 1555570800,
        "last_renewed": 1524034800,
        "start": 1369258234
    },
    "is_premium": 0,
    "level": 0,
    "result": "success",
    "sku": "gold_monthly",
    "type": "basic",
    "user_id": 41,
    "is_trial_eligible": false
    "is_apple_auto_renewable": false
    "is_google_subscription": false
*/

@JsonSerializable
data class MembershipData(
    val date: ExpirationInfo = ExpirationInfo(),
    val is_premium: Int = 0, // it's complaining about this line, seemingly
    val level: Int = 0,
    val result: String = "",
    val sku: String = "",
    val type: String = "",
    val user_id: Int = 0,
    val is_trial_eligible: Boolean = false,
    val is_apple_auto_renewable: Boolean = false,
    val is_google_subscriber: Boolean = false
) 

@JsonSerializable
data class ExpirationInfo(
    val expires: Long = 0,
    val last_renewed: Long = 0,
    val start: Long = 0
)
```

I will note that a very similar structure works for other data classes, without issue. Any ideas? I'm using **v1.0.2**

--- a/compiler/src/main/kotlin/se/ansman/kotshi/VariableElementExt.kt
+++ b/compiler/src/main/kotlin/se/ansman/kotshi/VariableElementExt.kt
@@ -1,17 +1,10 @@
 package se.ansman.kotshi
 
-import com.squareup.javapoet.TypeName
 import javax.lang.model.element.VariableElement
 
-fun VariableElement.getGetterName(): String {
-    // TODO: Don't convert it to a TypeName before comparing
-    var typeName = asType().asTypeName()
-    if (typeName.isBoxedPrimitive) {
-        typeName = typeName.unbox()
-    }
-    return if (typeName == TypeName.BOOLEAN && simpleName.startsWith("is")) {
+fun VariableElement.getGetterName(): String =
+    if (simpleName.startsWith("is")) {
         simpleName.toString()
     } else {
         "get${simpleName.toString().capitalize()}"
     }
-}

--- a/tests/src/main/kotlin/se/ansman/kotshi/ClassWithWeirdNames.kt
+++ b/tests/src/main/kotlin/se/ansman/kotshi/ClassWithWeirdNames.kt
@@ -10,5 +10,8 @@ data class ClassWithWeirdNames(
     val adapter2: Int?,
     val types: List<String>,
     val moshi: String,
-    val isStuff: Int
+    val isStuff: Int,
+    val is_stuff: Int,
+    val getStuff: Int,
+    val get_stuff: Int
 )

--- a/tests/src/main/kotlin/se/ansman/kotshi/ClassWithWeirdNames.kt
+++ b/tests/src/main/kotlin/se/ansman/kotshi/ClassWithWeirdNames.kt
@@ -9,5 +9,6 @@ data class ClassWithWeirdNames(
     val adapter1: Byte,
     val adapter2: Int?,
     val types: List<String>,
-    val moshi: String
+    val moshi: String,
+    val isStuff: Int
 )

